src/test/java/de/tum/in/www1/artemis/TextExerciseIntegrationTest.java

```diff
@@ -107,6 +110,26 @@ public void deleteExamTextExercise() throws Exception {
         assertThat(textExerciseRepository.findAll()).isEmpty();
     }
 
+    @Test
+    @WithMockUser(value = "tutor1", roles = "INSTRUCTOR")
```

Comment:
For the sake of consistency, I would change the value to "instructor1" (or do we often test with tutors that have instructor roles?)

Reply:
as far as I know the value does not really have an effect, the important thing is the role. But, I can change "value" to keep the consistency.

Reply:
yes, it does not affect the outcome, just for consistency ;)